probables/blooms/bloom.py

```diff
@@ -321,10 +355,10 @@ def __load(self, filepath: Union[str, Path], hash_function: Union[HashFuncT, Non
         # read the file, set the optimal params
         # mmap everything
         with open(filepath, "r+b") as filepointer:
-            offset = calcsize("QQf")
+            offset = self.CNT_FOOTER_STUCT.size
             filepointer.seek(offset * -1, os.SEEK_END)
-            mybytes = unpack("QQf", filepointer.read(offset))
-            vals = super(BloomFilterOnDisk, self)._set_optimized_params(mybytes[0], mybytes[2], hash_function)
+            mybytes = self.CNT_FOOTER_STUCT.unpack_from(filepointer.read(offset))
```

Comment:
`mybytes[0], mybytes[2]` are not semantic enough names. How about a destructuring assignment?